### ISSUE TITLE
Create none macOS only design assets

### ISSUE THREAD

**Issue by MikeCodesDotNET** [Microsoft/app-innovation-workshop#15]
It'd be great to have our design assets available to none macOS users. We could support the following: 

* Adobe XD 
* Figma

**Comment by robinmanuelthiel** [comment]
With a tear in one eye (I love Sketch), I support that!

**Comment by robinmanuelthiel** [comment]
Also, Figma is free until a specific usage! Which is Adobe XD not, correct?

**Comment by MikeCodesDotNET** [comment]
Correct, XD is part of the Adobe Creative suit which isn't free. 

Lets create an epic for this as it'll likely need a bit of investigation and tidying up to get right.

**Comment by robinmanuelthiel** [comment]
I agree on not using any platform exclusive stuff, which is why we switched over to Visual Studio Code. Also, I would try to make the workshops as accessible as possible for everyone, which means using free tools. On the other hand, the Design Resources are not meant for participants and maybe should not be in the repo at all. Let's have a private chat aside about that!
